src/components/Popups/ReplyField/__tests__/ReplyField-test.tsx

```diff
@@ -1,36 +1,39 @@
 import React from 'react';
+import { EditorState } from 'draft-js';
 import { shallow, ShallowWrapper } from 'enzyme';
-import ReplyField, { Props } from '../ReplyField';
+import ReplyField, { Props, State } from '../ReplyField';
 
 describe('components/Popups/ReplyField', () => {
     const defaults: Props = {
-        className: 'ba-Popup-text',
-        disabled: false,
+        className: 'ba-Popup-field',
+        cursorPosition: 0,
+        isDisabled: false,
         onChange: jest.fn(),
         onClick: jest.fn(),
+        setCursorPosition: jest.fn(),
         value: '',
     };
 
-    const getWrapper = (props = {}): ShallowWrapper => shallow(<ReplyField {...defaults} {...props} />);
+    const getWrapper = (props = {}): ShallowWrapper<Props, State, ReplyField> =>
+        shallow(<ReplyField {...defaults} {...props} />);
 
-    describe('event handlers', () => {
-        test('should call onChange event', () => {
+    describe('render()', () => {
+        test('should render the editor with right props', () => {
             const wrapper = getWrapper();
-            const textarea = wrapper.find('textarea');
-
-            textarea.simulate('change', 'test');
-            textarea.simulate('click', 'test');
 
-            expect(defaults.onChange).toHaveBeenCalledWith('test');
-            expect(defaults.onClick).toHaveBeenCalledWith('test');
+            expect(wrapper.prop('className')).toBe('ba-Popup-field ba-ReplyField');
         });
     });
 
-    describe('render()', () => {
-        test('should render the textarea with right props', () => {
+    describe('handleChange()', () => {
+        test('should call onChange', () => {
             const wrapper = getWrapper();
 
-            expect(wrapper.prop('className')).toBe('ba-TextArea ba-Popup-text');
+            wrapper.instance().handleChange(({
+                getCurrentContent: jest.fn().mockReturnValue({ getPlainText: jest.fn().mockReturnValue('test') }),
+            } as unknown) as EditorState);
```

Comment:
Can we break the mock editor state out into a separate variable for readability?